packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -581,19 +556,23 @@ describe('authentication', () => {
     expect(message).not.toBeNull();
 
     // Wait 100ms for the message queue and 200ms for the close delay
-    setTimeout(() => {
-      expect(closeWindowSpy).toHaveBeenCalled();
-    }, 301);
+    await new Promise<void>(resolve =>
+      setTimeout(() => {
+        expect(closeWindowSpy).toHaveBeenCalled();
+        resolve();
+      }, 350),
```

Comment:
Do we really need 350ms timeouts? This kind of pattern can really add to the runtime of the unit tests (and if we are waiting a certain amount of time, are these tests actually reliable?)

Reply:
Hmm, this test didn't work with the 301ms timeout, so there's not a lot of room to lower that number, but I could see getting rid of this check altogether since the important part of this test is that closeWindow isn't called before the message comes through

Reply:
I would maybe leave these for now and open a bug and assign it to me to figure out why we need them the sleeps at all

Reply:
Sure, opened bug 5790851 for this